eHealthAfrica/aether
# feat: (kong) modularize kong configuration

#### What does this PR do?
Adds a `kongfig.yml` file to all core modules for module-level kong elements registration

#### How to test
Modify `kongfig.yml` file on any of the core modules `kernel, odk, couchdb-sync`
Run `docker-compose -f  ./docker-compose.yml up --build`
Check that the apis, plugins, consumers are correctly registered on kong `http://api.aether.local:8001`

#### Related JIRA Ticket
[AET-167](https://jira.ehealthafrica.org/browse/AET-167)
